lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVM.cpp

```diff
@@ -392,12 +392,17 @@ static Value storeShared(ConversionPatternRewriter &rewriter, Location loc,
   unsigned bits = val.getType().getIntOrFloatBitWidth();
   const char *c = bits == 64 ? "l" : (bits == 16 ? "h" : "r");
 
+#if USE_ROCM
```

Comment:
@weihanmines `ctx`, `bits`, `c` are not used in `USE_ROCM` path either. The macro can be moved around to depict that.

Reply:
moved bits and char pointer c to the end of storeShared Op in the latest PR.